internal/tools/aws/database_multitenant.go

```diff
@@ -957,7 +960,7 @@ func (d *RDSMultitenantDatabase) removeMigratedInstallationFromMultitenantDataba
 	return nil
 }
 
-func (d *RDSMultitenantDatabase) dropDatabase(rdsClusterID, rdsClusterendpoint string, logger log.FieldLogger) error {
+func (d *RDSMultitenantDatabase) cleanupDatabase(rdsClusterID, rdsClusterendpoint, username string, logger log.FieldLogger) error {
```

Comment:
I think we can actually get username inside `cleanupDatabase` function to reduce some repetition.
```suggestion
func (d *RDSMultitenantDatabase) cleanupDatabase(rdsClusterID, rdsClusterendpoint string, logger log.FieldLogger) error {
    username := MattermostMultitenantDatabaseUsername(d.installationID)
```